setup.cfg

```diff
@@ -1,24 +1,3 @@
-[tool:pytest]
-norecursedirs =
-    .git
-    .tox
-    dist
-    env
-    build
-    venv
-testpaths =
-    src
-    tests
-python_files =
-    test_*.py
-doctest_plus = disabled
-addopts = --strict-markers
-markers =
-    slow
-    remote_data
-    filterwarnings
-    mpl_image_compare
-
 [flake8]
```

Comment:
I see that `flake8` does not support `pyproject.toml` and probably it will never do https://github.com/PyCQA/flake8/issues/234#issuecomment-812800722 I understand the reasoning, but it's a pity anyway